articles/aks/cluster-container-registry-integration.md

```diff
@@ -51,6 +51,30 @@ $MYACR = 'myContainerRegistry'
 
 New-AzContainerRegistry -Name $MYACR -ResourceGroupName myContainerRegistryResourceGroup -Sku Basic
 ```
+#### [Terraform](#tab/terraform)
+
+```terraform
+# Example Usage
+
+resource "azurerm_container_registry" "acr" {
+  name                = "containerRegistry1"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  sku                 = "Premium"
+  admin_enabled       = false
+  georeplications {
+    location                = "East US"
+    zone_redundancy_enabled = true
+    tags                    = {}
+  }
+  georeplications {
+    location                = "North Europe"
+    zone_redundancy_enabled = true
+    tags                    = {}
+  }
+}
+```
+> **_NOTE_** [Ref]: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/container_registry
```

Comment:
This syntax is not consistent with our formatting standard for our note blocks. Also, this information would be better delivered as a sentence or two. I'm going to create a work item and submit a PR against our private repo to incorporate your proposed changes into the article, along with some formatting and other minor changes to align with our writing style and standards.